erizo/src/erizo/rtp/SenderBandwidthEstimationHandler.h

```diff
@@ -25,7 +25,7 @@ class SenderBandwidthEstimationHandler : public Handler,
 
  public:
   static const uint16_t kMaxSrListSize = 20;
-  static const uint32_t kStartSendBitrate = 300000;
+  static const uint32_t kStartSendBitrate = 1000000;
```

Comment:
too high, I think to remember 300k is the same value Chrome uses inside by default, so there's no need to use a different default value. We might add a config value to change it.